install.json

```diff
@@ -0,0 +1,121 @@
+[ {
+  "id" : "mod-agreements-1.2.0",
```

Comment:
Sooooo, I'm not sure if it's relevant but if it is...and don't shoot the messenger because that's _all I am here_...the `mod-agreements` repo doesn't actually create a `mod-agreements` module. It creates an `olf-erm` module. So it'd actually be `olf-erm-1.2.0`. 

<sub><sup>I don't like it either but 🤷‍♂️ </sub></sup>

Reply:
We actually have this module's docker artifact created as "mod-agreements" at: https://cloud.docker.com/u/folioorg/repository/docker/folioorg/mod-agreements